scripts/helpers/general.sh

```diff
@@ -14,6 +14,15 @@ fi
 
 # Execution helpers; necessary for BATS testing and log output in buildkite
 
+function printdt() {
+    echo "[`date -u '+%FT%T.%3NZ'`]$ $1"
```

Comment:
The `%3N` doesn't seem to be support for the MacOS version of bash, so I may need to drop the subseconds here or find a way that works for both versions.